.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -45,25 +47,33 @@ jobs:
           echo "========== check paths of modified files =========="
           git diff --name-only HEAD^ HEAD > files.txt
 
-          echo "::set-output name=run_job::false"
+          echo "::set-output name=run_platform::false"
+          echo "::set-output name=run_website::false"  
 
           while IFS= read -r file
           do
             echo $file
             if [[ $file == netmanager/* ]]; then
-              echo "::set-output name=run_job::true"
+              echo "::set-output name=run_platform::true"
+              break
+            fi
+
+            if [[ $file == website/* ]]; then
+              echo "::set-output name=run_website::true"
             fi
```

Comment:
There is a break at line 58, You might want to check whether rest if the code shall be executed once the break is encountered